src/sparse-merkle-tree/sparse-merkle-tree.js

```diff
@@ -47,9 +47,16 @@ function setNodeValue(db, key, value, prefix) {
 * @returns {Buffer} - Key node value
 */
 function getHashFinalNode(hi, hv) {
-  const hashArray = [hi, hv];
+  // const hashArray = [hi, hv];
   // const hashKey = mimc7.multiHash(hashArray, bigInt(1));
-  const hashKey = poseidon.multiHash(hashArray, bigInt(1));
+  // const hashKey = poseidon.multiHash(hashArray, bigInt(1));
+
+  // let toH = [];
+  // toH.push(bigInt(1));
+  // toH.push(hashArray);
+  const hashArray = [bigInt(1), hi, hv];
+  const hashKey = poseidon.multiHash(hashArray);
```

Comment:
`poseidon.multiHash` is changed for `poseidon.hash` in all the file but here
Although the result is the same :smile_cat:

Reply:
yes, right, forgot to update the rest. Now all are with multiHash, to easily switch from mimc to poseidon and viceversa